omod-1.9/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_9/ConceptReferenceTermResource1_9.java

```diff
@@ -163,9 +164,14 @@ protected PageableResult doSearch(RequestContext context) {
 		String query = context.getParameter("q");
 		if (query == null)
 			return new EmptySearchResult();
-		List<ConceptReferenceTerm> terms = cs.getConceptReferenceTerms(query, null, context.getStartIndex(),
+        String sourceUuid = context.getParameter("source");

```

Comment:
q is a standard request parameter in the rest API but source isn't, my understanding is that search that involves a none standard request param should be implemented as a SearchHandler for purposes of documentation and correctness